src/analysis/Environments/PauseButton.ts

```diff
@@ -36,4 +44,25 @@ export function PauseButton(props: PauseButtonProps): ReactNode {
         [makeMenuIcon('pause'), 'Pause']
       )
     : null;
-}
+};
+
+export const pauseableRuntimeStatuses: LeoRuntimeStatus[] = [
```

Comment:
Nit, but I had to look into the leo code to check if these statuses were matching, would it be useful to comment that this should match the pauseable statuses defined in the backend here? https://github.com/DataBiosphere/leonardo/blob/c2e9ef69a292833566ba0e8be18103ff24876d97/core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/runtimeModels.scala#L175

Reply:
I almost included this link too! probably a good idea tbh